packages/engine-server/src/utils/engine.ts

```diff
@@ -0,0 +1,142 @@
+import {
+  APIUtils,
+  ConfigUtils,
+  CONSTANTS,
+  DendronError,
+  DEngineClient,
+  ErrorFactory,
+  NoteProps,
+  NoteUtils,
+  RespV3,
+} from "@dendronhq/common-all";
+import fs from "fs-extra";
+import _ from "lodash";
+import path from "path";
+
+export function openPortFile({ fpath }: { fpath: string }): number {
+  return _.toInteger(_.trim(fs.readFileSync(fpath, { encoding: "utf8" })));
+}
+
+export class EngineUtils {
```

Comment:
can we rename this file to EngineUtils.ts? I know it's redundant given the path it's in, but it helps me tremendously when viewing code when the file name matches the class / functionality. Otherwise, when I see the file name I'll think this is some `DEngine` implementation.

Reply:
hmm, fine renaming this but do we want to have a broader conversation around this? this pattern is used frequently across the codebase. we should adopt one of the following:
- always use `utils/{x}`
- always use `utils/{x}Utils`
- always use `utils/{x}` unless it conflicts with an existing file in which case we apply `utils/{x}Utils`

Also - while we're on it, we should also have a conversation around casing since we use it inconsistently. when to uppercase vs lowercase. I think everything is fine as long as we're consistent. here is airbnb's style guide on naming as an example: https://github.com/airbnb/javascript#naming--filename-matches-export